### PR TITLE
Update readme to align with TUF

### PR DESCRIPTION
**Pull request by sajayantony**
The PR addresses #1684 and achieve the following
 
1. Call out that the repository is scoped to the TUF implementation. 
2. Remove the upper case 'Notary' in places where it can and reference the client and server components to avoid confusion with the 'Notary Project'

Please provide feedback since I'm doing this with the best context I have and there might be areas or discussions I have missed.

### CODE REVIEW

File: README.md
Comment (toddysm) [inline README.md:9]: I would replace "Notary" with `notary`

File: README.md
Comment (toddysm) [inline README.md:7]: What do we mean with "serves as an implementation"? Should we just say that "This repository contains the implementation of a client and a server components using TUF"?

File: README.md
Comment (toddysm) [inline README.md:25]: Should we change "Notary Server" to "`notary` server component"

File: README.md
Comment (toddysm) [inline README.md:34]: Should we use `notary` to make sure we mean the repository? Maybe change to "The `notary` client and server implement TUF"?

File: README.md
Comment (toddysm) [inline README.md:58]: Should we use `notary` instead?

**Review by toddysm — CHANGES_REQUESTED** [review]

File: README.md
Comment (sajayantony) [reply README.md:25]: Update to just `notary server`. Felt component was quite verbose.

File: README.md
Comment (iamsamirzon) [inline README.md:10]: I think it will help if we say 
- the year  ( was it 2016 ?) when notary was released, as that will help readers date it. 
- explicitly state that it does not follow the specifications called out in notaryproject/notaryproject repo. ( I think we are planning to archive this repo, and when we do, we can come optionally come back and change this)

**Review by iamsamirzon — COMMENTED** [review]
I think it will help if we say

the year ( was it 2016 ?) when notary was released, as that will help readers date it.
explicitly state that it does not follow the specifications called out in notaryproject/notaryproject repo. ( I think we are planning to archive this repo, and when we do, we can come optionally come back and change this)

File: README.md
Comment (toddysm) [reply README.md:10]: +1
Though, I am not sure what we mean with "released" in this case - there has never been 1.0.0 of the implementations. The last stable version is 0.6.1 from 2018 (i.e. 5 years ago), the first implementation was released in Nov 2015 (v0.1)

**Comment by sajayantony** [comment]
I will follow up with the current state and archive comment as another PR. 
I was hoping to have this focus on just calling out the pointers to TUF specification and clarify upper case Notary to `notary` which is this repository to unblock https://github.com/notaryproject/.github/issues/35

File: README.md
Comment (sajayantony) [reply README.md:10]: Need to add information for DCT and call out the first release as a follow up.

**Comment by iamsamirzon** [comment]
LGTM But IANAM

**Review by toddysm — APPROVED** [review]
LGTM

**Comment by sajayantony** [comment]
Could we have other @notaryproject/notaryproject-notary-maintainers please help review this?

**Comment by gokarnm** [comment]
Changing `Notary` to lowercase `notary` does not make this less confusing with respect to `Notary Project` repo. I'd prefer keeping `Notary` as is, and renaming 'Notary Project' to 'Notation Specifications' or similar. Another option is to rename `Notary Project` to 'Specifications' with 'Notation' as a sub directory

**Comment by sajayantony** [comment]
@gokarnm - The PR came about from a call to clarify in the "Notice" section that this was an implementation of TUF and there is a CLI and server. The change is scoped only to that. The reference to `notary` in this repo is a part of the larger issue and I'm hoping we can get the Notice section in this repo clarified and updated. This is just a very minor part of the uber issue - https://github.com/notaryproject/.github/issues/35

I think we can discuss the specification repo changes in - https://github.com/notaryproject/.github/issues/38

**Comment by toddysm** [comment]
@SteveLasker and @justincormack would you mind taking a look at this PR?

**Comment by sajayantony** [comment]
Pinging @notaryproject/notaryproject-notary-maintainers and this was brough up on the call.

**Review by SteveLasker — APPROVED** [review]
I'd suggest we should have some folks from the TUF community/maintainers comment. This is LGTM to me.

**Comment by sajayantony** [comment]
Thanks @SteveLasker. 
During the call this was brought up again and the action item was the follow up with tagging the @notaryproject/notaryproject-governance-maintainers to see if we can get feedback and/or merge this.

**Comment by justincormack** [comment]
I think some of the uses of lower case notary are a bit confusing - it doesnt really make it less confusing to be lower case. I am generally ok though.

**Comment by sajayantony** [comment]
I wonder if repo owners can merge PR since this is only a readme update and I don't think it has anything to do with the CI failures.

**Comment by yizha1** [comment]
> I wonder if repo owners can merge PR since this is only a readme update and I don't think it has anything to do with the CI failures.

@toddysm @SteveLasker do you have any suggestions?  The failures of CI jobs were not related to changes to `readme.md` file by this PR, and probably, the main reason could be the CI and `notary` code are not maintained for a long time. The last merged [PR](https://github.com/notaryproject/notary/commit/887a007da8841e49023684656926d79d89d5a9d8) in main branch was 9 months ago, and the last fix for the ci is one year ago, see https://github.com/notaryproject/notary/commit/4373c538239356e23363e29639fbf9e4977b03e4

- ci/circleci: job_01 failed at execution of "make ci && codecov"
- ci/circleci: job_02 failed at execution of "make ci && codecov"
- ci/circleci: job_03 failed at execution of "make lint"
- ci/circleci: job_04 failed at execution of "RethinkDB integration"

File: README.md
Comment (gokarnm) [inline README.md:7]: typo `implementaiton`

File: README.md
Comment (gokarnm) [inline README.md:8]: typo `specficaitions`

**Review by gokarnm — CHANGES_REQUESTED** [review]
LGTM, except for minor typos.

File: README.md
Comment (sajayantony) [reply README.md]: Thanks @gokarnm - Fixed

**Review by gokarnm — APPROVED** [review]
LGTM!

**Comment by yizha1** [comment]
This PR was listed in issue https://github.com/notaryproject/.github/issues/35#issuecomment-1642938826, which reached a two-thirds supermajority approval. This PR itself also reached required approvals. The repo maintainers were tagged last month for reviewing this PR and addressing the failure of checks, but there has been no response till now. Since the PR only addresses the readme.md file and per the discussion of [community meeting on Jul 24 PDT time](https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw?both#Meeting-Notes), it is agreed that this PR can be merged bypass the CI checkers.

**Pull request merged**
